assets/index.js

```diff
@@ -58,20 +58,11 @@ const board = {
         {
           id: 9,
           title: 'Card title 9',
-          description: 'Card content'
-        }
-      ]
-    }
-  ]
+          description: 'Card content',
+        },
+      ],
+    },
+  ],
 }
 
-render(
```

Comment:
Can you please revert this file? I use those callbacks for dev purposes.

Reply:
Your local eslint rules are being applied to this project

Reply:
@lourenci If this file is also used for development purposes, can I correct the callbacks then and add another one like the rename? I thought it was only used for the Cypress test

Reply:
It's used to `yarn start` too.

I think you don't need to go through this.

Reply:
Okay, I'll just correct the renaming, so that I can check the style in a more practical way in this issue, adding the `allowRenameColumn` prop.

Reply:
What do you think @lourenci ?

Reply:
Ok, go for it.

Reply:
Any reason why this file doesn't have a complete usage example like the one in Codesandbox? I believe they could be the same, it would be interesting to test the usage manually with `yarn start`.